### DISCUSSION TITLE
`.ldf_` and `.cdf_` and their column labels

### DISCUSSION THREAD

**Discussion by kennethshsu** [casact/chainladder-python#185] [General]
```
genins = cl.load_sample("genins")
dev = cl.Development().fit(genins)
dev.ldf_
dev.cdf_
```

In the `.ldf_` call, the last column is titled 108-120, whereas in the `.cdf_` call, the last column is titled 108-ult instead. Is this intended?

**Comment by kennethshsu** [comment]
I think it makes sense, but wanted to make sure that the `.cdf_` tails the tail into consideration, whereas `.ldf_` does not?

**Comment by jbogaardt** [comment]
This is the intended behavior. On the `cdf_`, all labels are Age-Ult, e.g. 12-Ult, 24-Ult, etc. to denote that they are the cumulative of the available `ldf_`.  

At this point in the example, no tail estimator has been fitted (and they are optional).  Without a tail estimator, it is assumed that there is no tail (or more technically a tail = 1.0). If you do fit a tail, it will show up in the `cdf_` and `ldf_` vectors in the ways you'd expect.

Fun fact - you can to `dev.ldf_.incr_to_cum()` to calculate the `cdf_` as well. The `incr_to_cum` method recognizes that the `ldf_` triangle are a set of patterns.

**Comment by kennethshsu** [comment]
Ahh yes, I think I read it wrong the first time.

And what a neat feature with `dev.ldf_.incr_to_cum()`, I'll add that to the tutorial.
